src/containers/email/index.tsx

```diff
@@ -76,13 +79,37 @@ function neighborhoodToId(neighborhood: Neighborhoods): number {
 }
 
 const Email: React.FC = () => {
-  const [emailType, setEmailType] = useState<EmailType>(EmailType.INACTIVE);
+  const [emailType, setEmailType] = useState<string>(defaultTemplate);
+  const [templateNames, setTemplateNames] = useState<string[]>([]);
   const [filters, setFilters] = useState<EmailerFilters>(defaultFilters);
   const [fetchData, setFetchData] = useState<FilteredSite[]>([]);
   const [fetchSitesState, setFetchSitesState] = useState<LoadingState>(
     LoadingState.SUCCESS,
   );
   const [selectedEmails, setSelectedEmails] = useState<string[]>([]);
+  const [sendEmailForm] = Form.useForm();
+
+  function fetchTemplateNames() {
+    protectedApiClient
+      .getEmailTemplateNames()
+      .then((res) => {
+        setTemplateNames(res.templates);
+      })
+      .catch((err) => {
+        setTemplateNames([defaultTemplate]);
```

Comment:
⛏️ Can we add an error message here in the `catch` (same for `fetchTemplateData`? Something like the `message.error()` call in [this function ](https://github.com/Code-4-Community/speak-for-the-trees-frontend/blob/a9c31c362a83a278cdce864b9c92df18ed6ef61f/src/components/forms/sendEmailForm/index.tsx#L71) would do - if something fails on our end, it's better to explicitly tell the user that an error happened than to fail silently and have them guess at why something's not working